<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProblem</class>
 <widget class="QDialog" name="AddProblem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_ProblemName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Problem">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>题目名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>题目描述</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>样例输入</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_SamInput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>样例输出</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_SamOutput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Reset">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>610</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Accept">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>610</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancle">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>610</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_StdOutput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>标准输出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(如果题目没有样例输入输出，请只添加 标准输出 作为判题依据）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(题目描述中应当指出输出格式细节，包括空格以及换行的使用等)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
